website/content/en/docs/reference/configuration/template-syntax.md

```diff
@@ -102,23 +105,12 @@ Vector doesn't currently support fallback values. [Issue 1692][1692] is open to
 ### Missing fields
 
 If a field is missing, an error is logged and Vector drops the event. The `component_errors_total` internal
-metric is incremented with `error_type` = `template_failed`.
-
-### Nested fields
-
-Nested values can be accessed using Vector's [field notation syntax][paths]:
-
-```toml
-option = "{{ parent.child[0] }}"
-```
-
+metric is incremented with a `error_type` tag of `template_failed`.
```

Comment:
```suggestion
metric is incremented with an `error_type` tag of `template_failed`.
```

Reply:
Bah, originally I had written it as `incremented with a tag...` - thanks for the catch.